apps/www/pages/pricing/index.tsx

```diff
@@ -52,7 +52,7 @@ export default function IndexPage() {
       href: '#',
       from: true,
       priceMonthly: 25,
-      warning: 'per Project',
+      warning: 'per project',
```

Comment:
I think this looks better as all lowercase

Reply:
nice